tests/correctness/resources/miniboxing/tests/compile/type_classes_warnings.scala

```diff
@@ -0,0 +1,38 @@
+class ImplicitNumeric[@miniboxed G](implicit n: Numeric[G]) {}
+class ImplicitOrdering[@miniboxed G](implicit n: Ordering[G]) {}
+class ImplicitOrdered[@miniboxed G](implicit n: Ordered[G]) {}
+class ImplicitIntegral[@miniboxed G](implicit n: Integral[G]) {}
+class ImplicitFractional[@miniboxed G](implicit n: Fractional[G]) {}
+
+abstract class MyNumeric[@miniboxed Z] extends Numeric[Z] {}
+abstract class MyOrdering[@miniboxed Z] extends Ordering[Z] {}
+abstract class MyOrdered[@miniboxed Z] extends Ordered[Z] {}
+abstract class MyIntegral[@miniboxed Z] extends Integral[Z] {}
+abstract class MyFractional[@miniboxed Z] extends Fractional[Z] {}
```

Comment:
What if `G` and `Z` are not miniboxed? Please add a check that the plugin doesn't warn in this case.